File: LydianApi/Lydian.Entities/Models/CartItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lydian.Entities.Models
{
    public class CartItem
    {
        public int CartItemId { get; set; }
        public int CartId { get; set; }
        public int ProductId { get; set; }
        public Product Product { get; set; }
        public int Quantity { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

    }
}

File: LydianClient/src/app/models/cart-item.ts
import { Product } from './product';

export class CartItem{
  public cartItemId:number = 0;
  public cartId:number = 0;
  public productId:number = 0;
  public product:Product = new Product();
  public quantity:number = 0;
  public updatedAt:Date = new Date();

}

